tests/all.test.js

```diff
@@ -52,6 +52,15 @@ function runTests() {
         return Misc.JSONtoCSV([{ foo: "bar", baz: "qux" }]) === expected;
     });
 
+    test.assert("turns profiles into string[][]", () => {
+        const expected = [
+            [`foo`, `baz`, `third`],
+            [`bar`, `qux`, ``],
+            [`bar`, ``, `exists`]
+        ];
+        return JSON.stringify(Misc.profilesToCsvArray([{ foo: "bar", baz: "qux" }, {foo: "bar", third: "exists"}])) === JSON.stringify(expected);
+    });
```

Comment:
maybe add one with commas values in the data since you fixed that too? 😅 I guess you can just edit this data set to have 1/2 values with comma.